src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -4102,6 +4102,10 @@ class CgroupUtils(object):
                     pass
         # Give the OS a moment to update the tasks file
         time.sleep(0.1)
+        # After killing processes, tasks file could disappear when cgroup
+        # parent dir is cleaned up,
+        if not os.path.isfile(tasks_file):
+            return 0
```

Comment:
There is a very small window of time between checking whether the file exists and actually opening it. It's possible the directory could disappear in that small window. I suggest we place the call to open inside a try/except block and ignore the error if the file no longer exists.

Reply:
I agree. Ok, I'll just do the try...except on the second open().